app/views/students/show.html.erb

```diff
@@ -198,7 +198,9 @@
                   <% if !@tutor_options.nil? %>
                     <%= form_tag('set_tutor', method: 'put') do %>
                       <%= select_tag(:tutor_id, options_for_select(@tutor_options)) %>
-                      <%= submit_tag('Set Tutor') %>
+                      <%= button_tag(type: "submit", class: "btn btn-default") do %>
+                          Set Tutor <i class="icon-repeat"></i>
```

Comment:
We use [Bootstrap's glyphicons](https://getbootstrap.com/docs/3.3/components/#glyphicons) for icons. To add an icon use `<i class="glyphicon glyphicon-whatever"></i>`

Let's also try to use icons that help describe the action. For example, `glyphicon-remove` for deletions, `glyphicon-plus` for additions, etc